pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/Utils.groovy

```diff
@@ -270,7 +272,7 @@ public class Utils {
         return nodes
     }
 
-    static void markExecutedStagesOnAction(CpsScript script, String astUUID) throws Exception {
+    static void updateRunAndJobActions(CpsScript script, String astUUID) throws Exception {
```

Comment:
should be `@Restricted` but eh.

Reply:
I should do a whole chunk of that over in #210...

Reply:
And I did! =)